test.js

```diff
@@ -946,7 +916,7 @@ test('createAsyncIterator creates Iterator for Array', () => {
 })
 
 test('createAsyncIterator creates Iterator for Iterator', () => {
-  var myIterator = getIterator(['Alpha', 'Bravo', 'Charlie'])
+  var myIterator = getIterator((['Alpha', 'Bravo', 'Charlie']: Iterable<string>))
```

Comment:
My other theory on this is that an Array is an `Iterable` but it is also `mixed` but it not exactly either. And so when dealing with overloads, Flow is not sure which case to pick. Perhaps, flow should try to choose the strictest type possible in all cases. I'm trying to figure it out.